db/log_reader.cc

```diff
@@ -258,6 +258,10 @@ bool Reader::ReadRecord(Slice* record, std::string* scratch,
             //  writing a physical record but before completing the next; don't
             //  treat it as a corruption, just ignore the entire logical record.
             scratch->clear();
+          } else {
+            if (wal_recovery_mode == WALRecoveryMode::kPointInTimeRecovery) {
+              ReportOldLogRecord(scratch->size());
```

Comment:
Since this is the only place `ReportOldLogRecord` is called, it seems the method may not be needed - `reporter_->OldLogRecord` can be called directly.